### ISSUE TITLE
bmx6-auto-gw-mode needs to be conferted from bmx6 to bmx7

### ISSUE THREAD

**Issue by ilario** [libre-mesh/lime-packages#51]
In the watchping files of  bmx6-auto-gw-mode
```
lime-packages/packages/bmx6-auto-gw-mode/files/etc/watchping/wan-ok.d/bmx6-gw
lime-packages/packages/bmx6-auto-gw-mode/files/etc/watchping/wan-fail.d/bmx6-gw
```
 there are still references to bmx6 command which is not present anymore in develop (now just bmx7 executable is present).

**Comment by egon0** [comment]
i think i have fixed this in pr #54

**Comment by G10h4ck** [comment]
It seems to me that #54 fix this :)
